packages/db-postgres/src/transform/read/traverseFields.ts

```diff
@@ -443,6 +443,14 @@ export const traverseFields = <T extends Record<string, unknown>>({
                 relationships,
                 table,
               })
+
+              field.fields.forEach((field) => {
+                if ('name' in field && Boolean(field?.name)) {
+                  const fieldToDelete = `${groupFieldPrefix}${field.name}`
+
+                  deletions.push(() => delete ref[fieldToDelete])
```

Comment:
As I read this I realize that what we talked about here isn't quite right.

Instead of having the loop on the fields at this iteration, we should be deleting the fields if there is a `fieldPrefix` when we iterate on the field.

If we do it this way we can be sure that fields nested in rows and collapsibles are also deleted from the parent. The way this PR is written we wouldn't be catching those.